src/pages/EntityDetails/Model/ModelTabs/ModelTabs.test.tsx

```diff
@@ -0,0 +1,196 @@
+import { fireEvent, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { vi } from "vitest";
+
+import { JIMMRelation, JIMMTarget } from "juju/jimm/JIMMV4";
+import type { RootState } from "store/store";
+import { jujuStateFactory, rootStateFactory } from "testing/factories";
+import {
+  generalStateFactory,
+  configFactory,
+  controllerFeaturesFactory,
+  controllerFeaturesStateFactory,
+  authUserInfoFactory,
+} from "testing/factories/general";
+import {
+  modelListInfoFactory,
+  rebacRelationFactory,
+} from "testing/factories/juju/juju";
+import {
+  modelFeaturesStateFactory,
+  modelFeaturesFactory,
+} from "testing/factories/juju/juju";
+import {
+  modelWatcherModelDataFactory,
+  modelWatcherModelInfoFactory,
+} from "testing/factories/juju/model-watcher";
+import { renderComponent } from "testing/utils";
+import { ModelTab } from "urls";
+
+import ModelTabs from "./ModelTabs";
+import { Label } from "./types";
+
+describe("ModelTabs", () => {
```

Comment:
These tests have been moved from the EntityDetails tests, with a few improvements.